src/masternodes/rpc_masternodes.cpp

```diff
@@ -39,7 +39,14 @@ UniValue mnToJSON(uint256 const & nodeId, CMasternode const& node, bool verbose,
 
         // Only get targetMultiplier for active masternodes
         if (node.IsActive()) {
-            obj.pushKV("targetMultiplier", pos::CalcCoinDayWeight(Params().GetConsensus(), node, GetTime(), ChainActive().Height()).getdouble());
+            auto stakerBlockTime = pcustomcsview->GetMasternodeLastBlockTime(node.operatorAuthAddress, node.creationHeight);
```

Comment:
The GetMasternodeLastBlockTime using createHeight should also change to the current block height?

Reply:
Yes. That was the bug in #505 as well, thanks.